apps/client/src/config/autofill.js

```diff
@@ -6,7 +6,7 @@ const getDataNeed = (checker) =>
 export const getDataNeedPageOrNext = (checker, autofillRoutes, routes) =>
   getDataNeed(checker)
     ? autofillRoutes[getDataNeed(checker)][0]
-    : routes.address;
+    : routes.wrapper;
```

Comment:
I think you need to pull origin/chappie2.0 because this is already in there

Reply:
ah ok.